src/Service/QuestionInstanceService.php

```diff
@@ -86,6 +95,20 @@ public function createQuestionInstances(QuizTemplate $quizTemplate, QuizInstance
         }
     }
 
+    /**
+     * Highlights the answers of the code type questions.
+     *
+     * @param QuestionInstance $question
+     * @param AnswerInstance $answer
+     */
+    private function getHighlightCodeAnswer(QuestionInstance $question, AnswerInstance $answer)
```

Comment:
return type `void`

Reply:
name is also a problem here, a method that is named `get` and doesn't return anything is misleading.
maybe change the name to `highlightCodeAnswer`